agent/containermetadata/write_metadata.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Most of my writing operations was originally here and has since been moved to other places. It is probably better to name it utils.go